packages/cli/templates/form-trigger-completion.handlebars

```diff
@@ -73,9 +75,40 @@
 				</section>
 			</div>
 		{{/if}}
-		<script>
-			fetch('', { method: 'POST', body: {}, }).catch(() => {});
-		</script>
+		{{#if redirectUrl}}
+			<a id='redirectUrl' href='{{redirectUrl}}' style='display: none;'></a>
+		{{/if}}
+	<script>
+			fetch('', {
```

Comment:
could we not submit a html form, and let the http headers take care of the redirect here?

Reply:
this for consistency with form-trigger.handlebars